ethereum_history_api/circuits/lib/src/account.nr

```diff
@@ -3,14 +3,15 @@ use dep::std::println;
 use dep::u2b::{u64_to_u8, u120_to_u8};
 use crate::misc::arrays::{alter_array, sub_array_equals_up_to_length};
 use crate::HASH_LENGTH;
+use crate::misc::types::Bytes32;
 
 global PROOF_LEN = 4256;
 
 struct Account {
     nonce: u64,
     balance: u120,
```

Comment:
Let's add Bytes8 and Bytes15 for the types above

Reply:
done